src/cmd/devdraw/cocoa-screen-metal.m

```diff
@@ -373,6 +374,12 @@ - (BOOL)applicationShouldTerminateAfterLastWindowClosed:(NSApplication *)theAppl
 	return YES;
 }
 
+- (void)windowWillClose:(NSNotification *)notification
```

Comment:
Does this need a notification registered for it, too? Or is this enabled by default when asking for another notification on the window?